locales/es.json

```diff
@@ -29,35 +29,35 @@
-  "footer-col-1-3": "<a href='https://www.freecodecamp.org/news/if-name-main-python-example/'>Python Main</a>",
+  "footer-col-1-1": "<a href='https://www.freecodecamp.org/espanol/news/como-clonar-una-branch-en-git/'>Git Clone</a>",
+  "footer-col-1-2": "<a href='https://www.freecodecamp.org/espanol/news/metodologia-agile/'>Métodos Agile</a>",
+  "footer-col-1-3": "<a href='https://www.freecodecamp.org/espanol/news/python-if-name-main/'>Python Main</a>",
   "footer-col-1-4": "<a href='https://www.freecodecamp.org/news/javascript-callback-functions-what-are-callbacks-in-js-and-how-to-use-them/'>Callback</a>",
-  "footer-col-1-5": "<a href='https://www.freecodecamp.org/news/debounce-javascript-tutorial-how-to-make-your-js-wait-up/'>Debounce</a>",
-  "footer-col-1-6": "<a href='https://www.freecodecamp.org/news/javascript-url-encode-example-how-to-use-encodeuricomponent-and-encodeuri/'>URL Encode</a>",
+  "footer-col-1-5": "<a href='https://www.freecodecamp.org/espanol/news/curso-debounce-javascript-como-hacer-que-tu-js-espere/'>Debounce</a>",
+  "footer-col-1-6": "<a href='https://www.freecodecamp.org/espanol/news/url-codificacion-como-utilizar-encodeuricomponent-javascript/'>URL Encode</a>",
   "footer-col-1-7": "<a href='https://www.freecodecamp.org/news/make-it-blink-html-tutorial-how-to-use-the-blink-tag-with-code-examples/'>Blink HTML</a>",
-  "footer-col-1-8": "<a href='https://www.freecodecamp.org/news/python-returns-multiple-values-how-to-return-a-tuple-list-dictionary/'>Python Tuple</a>",
-  "footer-col-1-9": "<a href='https://www.freecodecamp.org/news/javascript-array-insert-how-to-add-to-an-array-with-the-push-unshift-and-concat-functions/'>JavaScript Push</a>",
+  "footer-col-1-8": "<a href='https://www.freecodecamp.org/espanol/news/tuplas-listas-python/'>Python Tupla</a>",
+  "footer-col-1-9": "<a href='https://www.freecodecamp.org/espanol/news/como-agregar-elementos-a-un-arreglo/'>JavaScript Push</a>",
   "footer-col-1-10": "<a href='https://www.freecodecamp.org/news/java-list-tutorial-util-list-api-example/'>Java List</a>",
-  "footer-col-2-1": "<a href='https://www.freecodecamp.org/news/learn-ux-design-self-taught-user-experience-designer/'>UX</a>",
-  "footer-col-2-2": "<a href='https://www.freecodecamp.org/news/what-is-design-thinking-an-introduction-to-the-design-process-for-entrepreneurs-and-developers/'>Design Thinking</a>",
-  "footer-col-2-3": "<a href='https://www.freecodecamp.org/news/prime-numbers-list-chart-of-primes/'>Prime Numbers List</a>",
-  "footer-col-2-4": "<a href='https://www.freecodecamp.org/news/product-design-explained-in-plain-english/'>Product Design</a>",
+  "footer-col-2-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-ux-convertirse-en-un-disenador-de-experiencia-de-usuario/'>UX</a>",
+  "footer-col-2-2": "<a href='https://www.freecodecamp.org/espanol/news/pensamiento-de-diseno-emprendedores-desarrolladores/'>Pensamiento de Diseño</a>",
```

Comment:
Not sure if it's possible to shorten, but "Pensamiento de Diseño" is a bit too long for one line.

Reply:
This one is hard one to condensed

Reply:
Yeah, that's true. I'd like to get design thinking in there, but it's not possible with the current footer setup.

Would another term like "Proceso de Diseño" work for this article? I tried it locally and it seems to fit.

Reply:
Yep, this is good choice.